doc/api/test.md

```diff
@@ -298,20 +298,20 @@ describe.only('a suite', () => {
 
 ## Filtering tests by name
 
-The [`--test-name-pattern`][] command-line option can be used to only run tests
-whose name matches the provided pattern. Test name patterns are interpreted as
-JavaScript regular expressions. The `--test-name-pattern` option can be
-specified multiple times in order to run nested tests. For each test that is
-executed, any corresponding test hooks, such as `beforeEach()`, are also
-run. Tests that are not executed are omitted from the test runner output.
-
-Given the following test file, starting Node.js with the
-`--test-name-pattern="test [1-3]"` option would cause the test runner to execute
-`test 1`, `test 2`, and `test 3`. If `test 1` did not match the test name
-pattern, then its subtests would not execute, despite matching the pattern. The
-same set of tests could also be executed by passing `--test-name-pattern`
-multiple times (e.g. `--test-name-pattern="test 1"`,
-`--test-name-pattern="test 2"`, etc.).
+The [`--test-name-pattern`][] and [`--test-skip-pattern`][] command-line
+options allow selecting which tests to run and which to
+skip based on their names.
+
+### Using `--test-name-pattern`
+
+The `--test-name-pattern` option filters tests based on their names by
```

Comment:
There is a lot of subtle unrelated doc rewriting happening in these sections. Can you undo them and propose them as a separate docs PR to keep this PR easier to review.

Reply:
Unresolving this, there is still a lot of unrelated documentation changes